src/utils/notifications.ts

```diff
@@ -72,6 +72,27 @@ export const handlePushManagerUnsubscribe = async (hide: () => void) => {
   }
 };
 
+export const manageNotification = async (accountId: string, notificationType: string, block: boolean) => {
+  const serviceWorker = await navigator.serviceWorker.ready;
+  const subscription = await serviceWorker.pushManager.getSubscription();
+  const endpoint = '/preferences/set';
+
+  const data = {
+    accountId: accountId,
+    endpoint: subscription?.endpoint,
```

Comment:
@erditkurteshiSQA are you sure `subscription?.endpoint` is the same as `endpoint`? In your previous implementation you've been used `'/preferences/set'` in two places: on this line and for `fetch`. Also I think we need to add `endpoint` as a function property I think to make this method more configurable. WDYT @erditkurteshiSQA ?

Reply:
There has been a misunderstanding on the first endpoint used.

Should be all fixed in the latest commit here:
https://github.com/near/near-discovery-components/pull/520